### PR TITLE
[TASK] Remove left hand menu

### PR DESCRIPTION
**Pull request by linawolf**
We have a complete menu of documentation on the top now.

Condense information and Cards to give a faster overview to newbies.

List the official references for experienced developers.

The start page will look like this:

image

Ps: I am planning to make an option to hide the left hand menu

### CODE REVIEW

**Review by garvinhicking — APPROVED** [review]
Let's try it out, what could possibly go wrong ;)

File: Documentation/Index.rst [hunk 1]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+        This guide introduces some of the core concepts of TYPO3, guides you
+        through installation, helps you to set up your first small project with
+        a custom template and gives you information on trou
+
+        ..  card-footer:: :ref:`Get started with TYPO3 <t3start:start>`
+            :button-style: btn btn-secondary stretched-link
+
+    ..  card:: Site package tutorial
+
+        This tutorial describes step by step how to come from your first
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:42]:
```suggestion
        This tutorial describes step by step how to create your first
```

File: Documentation/Index.rst [hunk 2]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+        site package or theme in TYPO3 from the scratch.
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:43]:
```suggestion
        site package or theme in TYPO3 from scratch.
```

File: Documentation/Index.rst [hunk 3]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+
+        ..  card-footer:: :ref:`Build your own site package <t3sitepackage:start>`
+            :button-style: btn btn-secondary stretched-link
+
+    ..  card:: Editors guide
+
+        Learn about how to log into the TYPO3 backend, how to work with the
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:50]:
```suggestion
        Learn about how you log into the TYPO3 backend, how to work with the
```

File: Documentation/Index.rst [hunk 4]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+        page tree, how to create and manage content, how to work with translations
+        and other useful tricks for editors.
+
+        ..  card-footer:: :ref:`Add content to a TYPO3 page <t3editors:start>`
+            :button-style: btn btn-secondary stretched-link
+
+    ..  card:: Download TYPO3
 
-..  include:: Home/_OverviewInclude.rst.txt
+        For information about the different versions of TYPO3 and its system
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:59]:
```suggestion
        For information about the different versions of TYPO3 and their system
```

File: Documentation/Index.rst [hunk 5]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
 
-Support is available via StackOverflow and Slack. Visit the `help page
-<https://typo3.org/help>`__ for more information.
+References
+==========
 
-For information about the different versions of TYPO3 and its system
-requirements, visit https://get.typo3.org.
+:ref:`TYPO3 explained <t3coreapi:api-overview>`
+    The main document for advanced developers and integrators, describes
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:74]:
```suggestion
    The main document for advanced developers and integrators. It describes
```

File: Documentation/Index.rst [hunk 6]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+    the API, coding guidelines, configuration options and security
+    guidelines. Introduces extension development to enhance TYPO3.
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:76]:
```suggestion
    guidelines. Introduces creating extension to enhance TYPO3.
```

File: Documentation/Index.rst [hunk 7]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+:doc:`TYPO3 Core changelog <changelog:Index#typo3-core-changelog>`
+    Learn what is new in TYPO3 and how to migrate away from deprecated
+    functionality.
+:ref:`Fluid ViewHelper reference <t3viewhelper:start>`
+    Complete reference of all Fluid ViewHelper provided by TYPO3. Used
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:81]:
```suggestion
    Complete reference of all Fluid ViewHelpers provided by TYPO3. Used
```

File: Documentation/Index.rst [hunk 8]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+    to improve your templates.
+:ref:`TypoScript reference <t3tsref:start>`
+    Complete reference of TypoScript settings and configuration options.
+    Useful to improve the frontend output of your site.
+:ref:`TSconfig rference <t3tsconfig:start>`
+    Configuration options to customize the TYPO3 backend.
+:ref:`TCA reference <t3tca:start>`
+    The complete reference of the Table Configuration Array (TCA) which
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:89]:
```suggestion
    Complete reference of the Table Configuration Array (TCA) which
```

File: Documentation/Index.rst [hunk 9]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+    is used to map between raw database tables and their representation
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:90]:
```suggestion
    maps raw database tables to
```

File: Documentation/Index.rst [hunk 10]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+    as database records in the TYPO3 backend and in Extbase.
```
Comment (sarahmccarthy123) [inline Documentation/Index.rst:91]:
```suggestion
    records in the TYPO3 backend and in Extbase.
```

File: Documentation/Index.rst [hunk 11]
```diff
@@ -12,30 +21,71 @@ Welcome to the official TYPO3 Documentation
...
+    :columns: 1
+    :columns-md: 2
+    :gap: 4
+    :card-height: 100
+
+    ..  card:: Getting started guide
+
+        This guide introduces some of the core concepts of TYPO3, guides you
+        through installation, helps you set up your first small project with
+        a custom template and gives you information on trou
```
Comment (linawolf) [inline Documentation/Index.rst:35]:
```suggestion
        a custom template and gives you information on troubleshooting.
```

**Review by sarahmccarthy123 — APPROVED** [review]
Looks good :-)

**Pull request merged**